lib/workers/repository/onboarding/branch/index.ts

```diff
@@ -54,10 +55,29 @@ export async function checkOnboardingBranch(
   // TODO #7154
   const branchList = [onboardingBranch!];
   if (onboardingPr) {
+    logger.debug('Onboarding PR already exists');
+
+    isModified = await isOnboardingBranchModified(config.onboardingBranch!);
+    // if onboarding branch is not modified, check if onboarding config has been changed and rebase if true
+    if (!isModified) {
+      const commit = await rebaseOnboardingBranch(
+        config,
+        onboardingPr.bodyStruct?.rawConfigHash
+      );
```

Comment:
This will constantly rebase if the platform doesn't support embedded html comments in PR's?

Reply:
Yes, unfortunately.
I think I need to store the configHash in onboardingCache and use it for comaprison.
WDYT?? @rarkins

Reply:
yes, we should probably ignore this feature there